lib/triangle.js

```diff
@@ -0,0 +1,15 @@
+function isTriangle(side1, side2, side3) {
+
+  if (side1 <= 0 || side2 <= 0 || side3 <= 0) {
+    return false
+  }
+  if ((side1 + side2) > side3 && (side2 + side3) > side1 && (side1 + side3) > side2) {
```

Comment:
Nice work! If you ever return boolean values for an `if` statement, you can just return the statement on the `if` line.

e.g., `return (side1 + side2) > side3 && (side2 + side3) > side1 && (side1 + side3) > side2`

No worries though, I've definitely done this before! 👍